specs/backlog.md

```diff
@@ -0,0 +1,92 @@
+# Creation of Backlog as a Ranked List
+
+We want to ensure that decisions about what to work on incorporate prioritization's agreed by the the team, product owners, our partners and customers. Specifically, we want to reach the point that:
```

Comment:
`prioritization's` should be plural, not possessive.

Reply:
resolved in doc